src/js/azdata/azdataQueryPlan.js

```diff
@@ -73,6 +84,63 @@ azdataQueryPlan.prototype.init = function(container, iconPaths)
         icons.push(iconName);
     }
 
+    ////////////////////////////////////////////////////////////////////////////////////
+    // temporary button logic to confirm zoom in and zoom out works.
+    
+    var buttons = document.createElement('div');
+	buttons.style.position = 'absolute';
```

Comment:
incorrect indent here too.  looks like we're missing tabs & spaces.  I think there is a vscode setting we can check-in to the repo to control the defaults for the project.